app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt

```diff
@@ -64,6 +67,12 @@ class BrowserFragment : WebFragment(), CursorEvent {
         initSession()
     }
 
+    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+        val view = super.onCreateView(inflater, container, savedInstanceState)
+        view?.browserOverlay?.setNavigationEventHandler(this)
```

Comment:
nit: I think you should be more aggressive about casting to non-null types – if view is null, the overlay won't be set and we won't crash but we won't know about the problem and we may get a lot of unhappy users not using the app because these buttons just suddenly stop working in some circumstances